_pkgdown.yml

```diff
@@ -10,3 +10,21 @@ development:
 authors:
   Kirill Müller:
     href: https://krlmlr.info
+
+
+reference:
+- title: Main functions
+  contents:
+  - '`bump_version`'
```

Comment:
Here and everywhere:

```suggestion
  - bump_version
```

pkgdown really could do better here...

Reply:
I think the default pkgdown behavior is for "Weirdly named functions"